Documentation/Localization.ru_RU/guides.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guides xmlns="https://www.phpdoc.org/guides" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd" links-are-relative="true">
+  <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+             project-home="https://docs.typo3.org/m/typo3/tutorial-getting-started/main/ru-ru/"
+             project-contact="https://typo3.slack.com/archives/C028JEPJL"
+             project-repository="https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted"
+             project-issues="https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted/issues"
+             edit-on-github-branch="main"
+             edit-on-github="TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted"
+             typo3-core-preferred="stable"/>
+  <project title="Getting Started"
+           release="main (development)"
+           version="main (development)"
+           copyright="since 2012 by the TYPO3 contributors"/>
+
+  <inventory id="t3start11" url="https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/ru-ru/"/>
```

Comment:
I just converted what was there in the Settings.cfg - what should it be replaced to?

Reply:
If t3start11 is really used in the Russian translation, change it to the English one. If we leave it it will produce an error on rendering as the URL is not available.

Reply:
We should take care of this in a follow up. It will cause a warning and the link not rendererd. No errors to be expected